cronet/src/test/java/io/grpc/cronet/CronetClientTransportTest.java

```diff
@@ -61,7 +65,7 @@ public void setUp() {
         new CronetClientTransport(
             streamFactory,
             new InetSocketAddress("localhost", 443),
-            "",
+            "test.example.com",
```

Comment:
nit/optional: make this string a static variable?

Reply:
Done.